### DISCUSSION TITLE
Gravity field changes

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#163] [Polls]
Does ACG think gravity field changes are instant or retarded or advanced?

Prior discussion
https://github.com/orgs/a-cosmology-group/discussions/82#discussioncomment-7129869

An article from Andre Assis on the subject
https://www.ifi.unicamp.br/~assis/Action-at-a-Distance-p45-56(1999).pdf

**Comment by bgreenman** [comment]
I'm not sure which of the three approaches is right.more research is needed on the topic

**Comment by bgreenman** [comment]
Thats the latest information we have.We need more refinement in gravity field changes research

**Comment by sahil5d** [comment]
More on Assis's work https://johnplaice.substack.com/p/english-translations-of-coulomb-ampere

Weber's formula (a classic 1/r^2 term with a velocity-dependent term and an acceleration-dependent term) is probably exactly accurate for force on matter. It also might enable total understanding of the nucleus (how like charges attract at very very small distance).

**Comment by bgreenman** [comment]
I looked at the article you have mentioned, and i feel the action at a distance concept should be investigated and studied. Assis's works are worth investigating

**Reply by sahil5d** [reply]
Great.

The phrase "action at a distance" seems to make people lose their minds, so maybe call it "action between all distances" or "the force relation between all things in the volume".

**Reply by bgreenman** [reply]
Action between all distances or the the force relation betwwen all things is a more explainable term
